src/BuildIntegration/Microsoft.NETCore.Native.Windows.props

```diff
@@ -80,14 +80,20 @@ See the LICENSE file in the project root for more information.
       <Output TaskParameter="ExitCode" PropertyName="_WhereCppTools"/>
     </Exec>
 
-    <Exec Condition="'$(_WhereCppTools)' != '0'" Command="&quot;$(MSBuildThisFileDirectory)findvcvarsall.bat&quot; x86_amd64" LogStandardErrorAsError="true" ContinueOnError="false" ConsoleToMSBuild="true">
-      <Output TaskParameter="ConsoleOutput" PropertyName="_CppToolsDirectory"/>
+    <Exec Condition="'$(_WhereCppTools)' != '0'" Command="&quot;$(MSBuildThisFileDirectory)findvcvarsall.bat&quot; amd64" LogStandardErrorAsError="true" ContinueOnError="false" ConsoleToMSBuild="true" StandardOutputImportance="Low">
+      <Output TaskParameter="ConsoleOutput" PropertyName="_FindVCVarsallOutput"/>
       <Output TaskParameter="ExitCode" PropertyName="_VCVarsAllFound"/>
     </Exec>
 
+    <ItemGroup Condition="'$(_VCVarsAllFound)' == '0'">
+      <LIBPATHS Include="$(_FindVCVarsallOutput.Split(`#`)[1].Split(';'))" />
```

Comment:
I would make this `<AdditionalNativeLibraryDirectories ...>` . There is enough confusion about LIB and LIBPATH already.